cyipopt/tests/unit/test_scipy_optional.py

```diff
@@ -131,6 +135,94 @@ def test_minimize_ipopt_jac_hessians_constraints_with_arg_kwargs():
     np.testing.assert_allclose(res.get("x"), expected_res, rtol=1e-5)
 
 
+@pytest.mark.skipif("scipy" not in sys.modules,
+                    reason="Test only valid if Scipy available.")
+@pytest.mark.parametrize('method', MINIMIZE_METHODS)
+def test_minimize_ipopt_jac_with_scipy_methods(method):
+    x0 = [0] * 4
+    a0, b0, c0, d0 = 1, 2, 3, 4
+
+    def fun(x, a=0, e=0, b=0):
+        assert a == a0
+        assert b == b0
+        fun.count += 1
```

Comment:
This counts the number of executions so we can confirm that the code is actually being executed.